deltacat/compute/compactor/compaction_session.py

```diff
@@ -640,3 +643,14 @@ def _execute_compaction_round(
         new_round_completion_info,
         rcf_source_partition_locator,
     )
+
+
+def compact_partition_from_request(
+    compact_partition_params: CompactPartitionParams,
+) -> Optional[str]:
+    """
+    Wrapper for compact_partition that allows for the compact_partition parameters to be
+    passed in as a custom dictionary-like CompactPartitionParams object.
+    :param compact_partition_params:
+    """
+    return compact_partition(**compact_partition_params)
```

Comment:
I think we would also need to pass in the non-keyword arguments here to match the function signature of compact_partition.